it/compare-generation.ps1

```diff
@@ -95,21 +95,21 @@ if ($HashString1 -eq $HashString2) {
     Exit 0
 }
 else {
-    Write-Error "The content of the folders is NOT identical"
-    $archivePath1 = Join-Path $rootPath -ChildPath "idempotency-folder1.zip"
-    $archivePath2 = Join-Path $rootPath -ChildPath "idempotency-folder2.zip"
+    Write-Host "The content of the folders is NOT identical"
+    $resultsFolder = Join-Path -Path $rootPath -ChildPath "idempotency-results"
+    $archivePath1 = Join-Path $resultsFolder -ChildPath "idempotency-folder1.zip"
+    $archivePath2 = Join-Path $resultsFolder -ChildPath "idempotency-folder2.zip"
```

Comment:
nit: this could be in the dev false block instead

Reply:
done